examples/transcripts.ipynb

```diff
@@ -736,7 +736,7 @@
     }
    ],
    "source": [
-    "event_corpus_map = seattle.download_most_recent_transcripts()\n",
+    "event_corpus_map = seattle.download_event_transcripts()\n",
```

Comment:
Should this be `download_transcripts()` ?
Or change cdp_instance.py to have `download_event_transcripts()` instead

Reply:
Hmm I think you are right. There is no point to name it `download_event_transcripts` when we only have `event` transcripts to begin with. Will change and make downstream changes as well.